ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1993,6 +1993,9 @@ objects:
           echo "Waiting for logs ..."
           wait
 
+          tar -czC /tmp/artifacts/must-gather -f /tmp/artifacts/must-gather.tar.gz . &&
+          rm -rf /tmp/artifacts/must-gather
```

Comment:
hrm, these really feel like they should be indented but hopefully this is temporary